.vitepress/config.ts

```diff
@@ -106,6 +110,16 @@ export default defineConfig({
       md.use(mdFootnote);
     },
   },
+
+  vue: {
+    template: {
+      compilerOptions: {
+        isCustomElement: (tag) => {
+          return ["chatlio-widget"].includes(tag.toLowerCase());
```

Comment:
usually it's suggested to just check for `-`, but in our case, I'd opt for more granular control